packages/sources/coinbase-prime/src/index.ts

```diff
@@ -1,13 +1,13 @@
 import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
 import { Adapter } from '@chainlink/external-adapter-framework/adapter'
 import { config } from './config'
-import { balance } from './endpoint'
+import { balance, wallets } from './endpoint'
 
 export const adapter = new Adapter({
```

Comment:
I think we can use PorAdapter here, as it will automatically set some useful config overrides that are typically needed for PoR adapters

```suggestion
export const adapter = new PoRAdapter({
```